Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.20-alpine
+
+WORKDIR /app
+RUN apk add --no-cache git
+RUN git clone https://github.com/lightninglabs/lightning-faucet.git .
+RUN go mod download
```

Comment:
Is this actually necessary? The `go build` commands should take care of resolving the deps.

Reply:
From the docs: 
> The go mod download command is useful mainly for pre-filling the module cache or for loading data to be served by a [module proxy](https://go.dev/ref/mod#glos-module-proxy).

https://go.dev/ref/mod#go-mod-download

Reply:
Will remove